oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreBuilder.java

```diff
@@ -608,13 +621,17 @@ public long getPrevDocumentCacheSize() {
         return memoryCacheSize * prevDocCachePercentage / 100;
     }
 
+    public long getPrevNoPropCacheSize() {
```

Comment:
Shouldn't this check feature toggle first ?

Reply:
good point - if feature toggle is off it should actually use `0`

Reply:
addressed in https://github.com/apache/jackrabbit-oak/pull/1779/commits/faa44dffb661a8776dcef7d335e239096e17fce4